### PR TITLE
integration: put key on watch target member for TestWatchWithRequireLeader

### LINKED ISSUE #6386: TestWatchWithRequireLeader: Timed out
Via semaphore https://semaphoreci.com/coreos/etcd/branches/master/builds/892

```
goroutine 55316 [chan receive, 6 minutes]:
github.com/coreos/etcd/clientv3/integration.TestWatchWithRequireLeader(0xc822e87320)
	/home/runner/workspace/src/github.com/coreos/etcd/gopath/src/github.com/coreos/etcd/clientv3/integration/watch_test.go:672 +0x1285
testing.tRunner(0xc822e87320, 0x19ed6f8)
	/usr/local/golang/1.6.3/go/src/testing/testing.go:473 +0xdd
created by testing.RunTests
	/usr/local/golang/1.6.3/go/src/testing/testing.go:582 +0xae3
```

### PR DESCRIPTION
**Pull request by heyitsanthony**
It's possible the put will not propagate to all members before removing quorum,
causing watches on the key to wait forever.

Fixes #6386

### CODE REVIEW

File: clientv3/integration/watch_test.go [hunk 1]
```diff
@@ -672,8 +672,11 @@ func TestWatchWithRequireLeader(t *testing.T) {
 	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 3})
 	defer clus.Terminate(t)
 
-	// something for the non-require leader watch to read as an event
-	if _, err := clus.Client(1).Put(context.TODO(), "foo", "bar"); err != nil {
+	// Put a key for the non-require leader watch to read as an event.
+	// The watchers will be on member[0]; put key through member[0] to
+	// ensure that it receives the update so watching after killing quorum
+	// is guaranteed to have the key.
+	if _, err := clus.Client(0).Put(context.TODO(), "foo", "bar"); err != nil {
```
Comment (xiang90) [inline clientv3/integration/watch_test.go]: define 0 as toWatch? and use toWatch at line 694 and 695?

**Comment by xiang90** [comment]
LGTM

File: clientv3/integration/watch_test.go (hunk 1 above)
Comment (heyitsanthony) [inline clientv3/integration/watch_test.go]: `liveClient := clus.Client(0)` instead of naming the member index?

Comment (xiang90) [inline clientv3/integration/watch_test.go]: sure.

**Pull request merged**